bin/mv.js

```diff
@@ -141,31 +138,118 @@ function loadProfile(name) {
     return config.getProfile(name);
 }
 
-function moveCronJob(profile, name, target) {
+function moveCronJob(profile, name, target, options) {
     return coroutine(function*() {
-        var cronJobs = yield profile.listCronJobs();
-
-        for (let job of cronJobs) {
-            if (job.name === name) {
-                let token = yield profile.getWebtask({
-                    name: target.name
-                });
-                yield target.profile.createCronJob({
-                    name: target.name,
-                    container: target.container,
-                    token: token.token,
-                    schedule: job.schedule
-                });
-                yield profile.removeCronJob({
-                    name: name
-                });
-                break;
+        var job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
             }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
```

Comment:
`name` is the source name, `options.verify` the source token to verify and `target.name` the target name.

I agree, this function signature must be improved.

During my tests, on first move `Webtask#token` and `CronJob#token` match. The same doesn't hold true for subsequent moves. This seems not to be a problem because the migration and subsequent schedule executions are successful (?)